include/af/surface.h

```diff
@@ -0,0 +1,1194 @@
+/*******************************************************
```

Comment:
@syurkevi I am guessing this is a mistake. This `surface.h` is a redundant file with all the functions in `image.h`. Please remove this duplicate file and add all suface and plot3 functions should be added in `include/af/graphics.h`.